mentoring/mentor-guide.md

```diff
@@ -0,0 +1,100 @@
+
+There are many ways you can be a mentor within the Kubernetes ecosystem. Listening and being generally helpful is an informal start, but we’ve drawn together some programs that will be impactful. These programs take the mentee's AND mentor's time into consideration. We know that it’s difficult to carve out steady time to mentor someone, and we appreciate that contributor to the Kubernetes community! The Bonus: we set up the communication framework for you already so your expectations are set when you are ready to dig into mentoring.
+
+You are serving two roles - coach and advisor. You are a problem solver that doesn’t have all of the answers but will know the best resources to go to get the guidance that is needed. You are helping to create a culture of continuous improvement, collaboration, and feedback.
+
+Check out our [mentoring programs](README.md) for more detailed descriptions. Please use [this form](https://goo.gl/forms/Y4MH6NGHe6OU5cZC3) to sign up to mentor.
+
+## Responsibilities and Expectations
+*for ALL programs*
+* Your goal is to be a guide and not a teacher. If you have the time to teach someone the fundamentals of Go, please do, but know that this isn’t required. In this scenario, you should be able to help identify the best resources for learning Go for the mentee. 
+* Issue Grooming. Most levels of the [membership ladder](/community-membership.md) will require some form of issue grooming from you. Examples:
+*  picking issues that don’t span multiple SIGs for new contributors
+*  or they do because they are learning to be a better approver
+*  labeling that will help them identify clear issues that need their attention
+* Make introductions to resources and people for the mentee - this is sponsorship which is now equally as important as mentorship.
+* Help contributor be productive.
+* Patience and empathy.  Through repetition, people learn, and as a mentor, we cannot expect a mentee to meet all of our expectations.  Through the removal of expectation and bias, and with a sprinkle of compassion, we can gain more patience.
+* Have some fun and learn together!
+
+
+### Responsibilities by each program
+*[Group mentoring](group-mentoring.md)*
+
+You’ll need to be:
+*  On Slack at least 1-2 hours a week fielding questions.
+*  Available for a group bi-weekly standup to check the progress of the mentees. 
+*  Helpful with building comradery amongst the group - this can be as simple as curating conversation in slack.
+
+Mentees will be asked to take responsibility for their growth and improvement as it's a semi-structured self-paced program; however, you are there to help guide them by sharing your methodologies, resources, tips, and experiences.
+
+*[The 1:1 Hour](the1-on-1.md)*
```

Comment:
If the other PR's commit is not inbound, this link will be broken.  Actually that one links to this.  So ideally they merge in very close succession.

Reply:
This is doc is pending right now